src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java

```diff
@@ -363,89 +126,41 @@ public void onTestSkipped(ITestResult result) {
+    if (LOG.isTestStarted()) {
+      LOG.stopLogging();
     }
   }
 
   @Override
-  public void onTestFailedButWithinSuccessPercentage(ITestResult result) {
-  }
-
-  @Override
-  public void onStart(ITestContext context) {
-    CommonUtils.createDirectory(screenDirPath);
-    imageCounter = 0;
-    DateFormat dateFormat = new SimpleDateFormat("yyyy/MM/dd HH:mm:ss");
-    Date date = new Date();
-
-    StringBuilder builder = new StringBuilder();
-    builder
-        .append("<html><style>"
-                + "table {margin:0 auto;}td:first-child {width:200px;}td:nth-child(2) {width:660px;}td:nth-child(3) "
-                + "{width:100px;}tr.success{color:black;background-color:#CCFFCC;}"
-                + "tr.warning{color:black;background-color:#FEE01E;}"
-                + "tr.error{color:black;background-color:#FFCCCC;}"
-                + "tr.step{color:white;background:grey}"
-                + "</style><head><meta http-equiv=\"content-type\" content=\"text/html; charset=UTF-8\">"
-                + "<style>td { border-top: 1px solid grey; } </style></head><body>"
-                + "<script type=\"text/javascript\" src=\"http://code.jquery.com/jquery-1.8.2.min.js\"></script>"
-                + "<p>Date: "
-                + DateTimeFormat.forPattern("yyyy-MM-dd'T'HH:mm:ssZZ").print(
-            DateTime.now(DateTimeZone.UTC))
-                + "</p>"
-                + "<p>Polish Time: "
-                + DateTimeFormat.forPattern("dd/MM/yyyy HH:mm:ss ZZ").print(
-            DateTime.now().withZone(
-                DateTimeZone.forTimeZone(TimeZone.getTimeZone("Europe/Warsaw")))) + "</p>"
-                + "<p>Browser: " + Configuration.getBrowser() + "</p>" + "<p>OS: "
-                + System.getProperty("os.name") + "</p>" + "<p>Testing environment: "
-                + new UrlBuilder().getUrlForWiki(Configuration.getWikiName())
-                + "</p>" + "<p>Testing environment: " + Configuration.getEnv() + "</p>"
-                + "<p>Tested version: " + "TO DO: GET WIKI VERSION HERE" + "</p>"
-                + "<div id='toc'></div>");
-    CommonUtils.appendTextToFile(logPath, builder.toString());
-    appendShowHideButtons();
-    try {
-      FileInputStream input = new FileInputStream("./src/test/resources/script.txt");
-      String content = IOUtils.toString(input);
-      CommonUtils.appendTextToFile(logPath, content);
-    } catch (IOException e) {
-      System.out.println("no script.txt file available");
-    }
-  }
-
-  private void appendShowHideButtons() {
-    String hideButton = "<button id=\"hideLowLevel\">hide low level actions</button>";
-    String showButton = "<button id=\"showLowLevel\">show low level actions</button>";
-    StringBuilder builder = new StringBuilder();
-    builder.append(hideButton);
-    builder.append(showButton);
-    CommonUtils.appendTextToFile(logPath, builder.toString());
-  }
+  public void onTestFailedButWithinSuccessPercentage(ITestResult result) {}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1186)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)